src/Dto/Request/Signalement/CoordonneesBailleurRequest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Dto\Request\Signalement;
+
+class CoordonneesBailleurRequest
+{
+    public function __construct(
```

Comment:
Voir PR https://github.com/MTES-MCT/histologe/pull/1832
Vérifier que le téléphone soit valide 
lib pour php https://github.com/giggsey/libphonenumber-for-php